360/watchapp/views.py

```diff
@@ -51,22 +192,19 @@ def register_view(request):
             error = "Passwords do not match."
         else:
             # Create a new user
-            user = User.objects.create_user(
```

Comment:
Again, keep first name and last name, delete full name. If there is ever a doubt on what to keep, keep main's code intact.